client/index.js

```diff
@@ -0,0 +1,245 @@
+import {
+  setupDailyControls,
+  setupStoreQuery,
+  setupUploadForm,
+  updateStatus,
+  updateUploads,
+  enableStoreControls,
+  setupIndexUploads,
+  disableIndexUploads,
+  disableStoreControls,
+  enableStoreQuery,
+  updateResponse,
+  enableIndexUploads,
+  updateUploadError,
+  disableStoreQuery,
+} from './dom.js';
+
+const apiURL = 'http://127.0.0.1:5000';
+
+window.addEventListener('DOMContentLoaded', () => {
+  fetchCapabilities().then((capabilities) => {
+    if (capabilities.daily === true) {
```

Comment:
This will be false if the Daily API key is not specified in the server environment.